selfdrive/car/volkswagen/values.py

```diff
@@ -294,6 +294,12 @@ class CAR(Platforms):
     chassis_codes={"A3"},
     wmis={WMI.VOLKSWAGEN_USA_CAR},
   )
+  VOLKSWAGEN_JETTA_SPORTWAGEN = VolkswagenPQPlatformConfig(
+    [VWCarDocs("Volkswagen Jetta Sportwagen TDI 2011-2014")],
+    VolkswagenCarSpecs(mass=1518, wheelbase=2.62, minSteerSpeed=50 * CV.KPH_TO_MS, minEnableSpeed=20 * CV.KPH_TO_MS),
```

Comment:
update wheelbase to be accurate for sedan
```suggestion
    VolkswagenCarSpecs(mass=1518, wheelbase=2.65, minSteerSpeed=50 * CV.KPH_TO_MS, minEnableSpeed=20 * CV.KPH_TO_MS),
```